use-cases-and-requirements.bs

```diff
@@ -0,0 +1,219 @@
+<pre class='metadata'>
+Title: Solid Use Cases and Requirements
+Shortname: your-spec
+Level: 1
+Status: w3c/ED
+Group: solid-cg
+URL: http://example.com/url-this-spec-will-live-at
+Editor: elf Pavlik
+Abstract: Use Cases and Requirements for Solid ecosystem.
+</pre>
+
+Introduction {#intro}
+=====================
+
+Issue: Needs explaination how proposals should clearly reference requirements which they satisfy. As well as requirements they do not satisfy due to some limitations.
```

Comment:
I think it's also important to provide some expectations around use case scope. We should focus mostly on lean and easy to understand use cases and requirement that don't drift too far away from practical use, so they can be as informative and referenceable as possible.